source/_patterns/02-molecules/branding/_branding.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * Block - Branding.
+ *
+ * url: String: the url the block directs to.
+ * site_logo: String: the logo provided by the app.
+ * site_name: String: the site name provided by the app.
+ * site_slogan: String: the site slogan provided by the app.
+ *
+ */
+#}
+
+{% set site_logo = {
+  svgpath: pl ? '@molecules/branding/logo.svg' : site_logo,
```

Comment:
Let's revisit this.

Reply:
Let's use the method I posted in the flow a week or so ago. It basically sets up all variables for demo purposes in PL if the variable array is not already being passed (such as from a source that includes/embeds/extends the pattern like a page that wants to pass its own data and not Faker/Lorem Ipsum).

```
{% if pl and not brandingBlock %}
  {% set brandingBlock = {
    variable1: value1,
    variable2: value2,
  } %}
{% endif %}
```

We then also have data encapsulation, by namespacing our variables. This avoids headaches later when passing variables to included patterns at a higher level. You'd then use these as:

`{{ brandingBlock.variable1 }}` etc.

Reply:
Awesome, on it!